studio/app/common/core/rules/runner.py

```diff
@@ -33,9 +33,15 @@ def run(cls, __rule: Rule, last_output):
 
             cls.set_func_start_timestamp(os.path.dirname(__rule.output))
 
+            nwb_params = cls.get_nwb_params(os.path.dirname(__rule.output))
+
```

Comment:
I think you don't have to get nwb params from experiment.yaml.
`nwbfile` in line27 already has them.